### PR TITLE
refactor: get rid of those damn "pb" alias

### PR DESCRIPTION
**Pull request by 3Hren**
Sorry, guys.

### CODE REVIEW

**Comment by 3Hren** [comment]
image

**Comment by antmat** [comment]
img

File: connor/antifraud/blacklist_watcher_test.go [hunk 1]
```diff
@@ -14,7 +14,7 @@ import (
 )
 
 // avoid using the mockgen because it cannot properly mock stream method
-// that are also declared in the proto/node.pb.go
+// that are also declared in the proto/node.sonm.go
```
Comment (antmat) [inline connor/antifraud/blacklist_watcher_test.go]: unnescescary replace

**Comment by 3Hren** [comment]
Done!

**Pull request merged**